File: cpp/n441_CountCoin.cpp
// https://leetcode-cn.com/problems/arranging-coins/
#include <algorithm>
#include <math.h>
using namespace std;

class Solution {
public:
    int arrangeCoins(int n) {
        return int(sqrt(double(2*(double)n + 0.25)) - 0.5);
    }
};

int main(){
    return 0;
}

File: python/n441_CountCoin.py
# https://leetcode-cn.com/problems/arranging-coins/

from math import sqrt, floor
class Solution:
    def arrangeCoins(self, n: int) -> int:
        return floor( sqrt(1/4+2*n)-1/2 )
        

if __name__ == "__main__":
    pass           
